oneview/resource_fcoe_network.go

```diff
@@ -166,6 +232,32 @@ func resourceFCoENetworkUpdate(d *schema.ResourceData, meta interface{}) error {
 		return fmt.Errorf("Initial scope uri can not be updated")
 	}
 
+	if d.HasChange("bandwidth") {
+		rawVal := d.Get("bandwidth")
+		bandwidthVal := rawVal.([]interface{})
+		for _, bandwidth := range bandwidthVal {
+			rawBandwidth := bandwidth.(map[string]interface{})
+			conTempURI := utils.NewNstring(d.Get("connection_template_uri").(string))
```

Comment:
line 240-257 is same across different network: Ethernet, FC, FCOE. It would have been better if we think of some common function to update the connection template bandwidth rather than repeating it multiple time. But I can see it would not give any benefit now...in future keep it in mind to utilize common function if the functionalities same across entities.